lib/perso.ml

```diff
@@ -2820,70 +2814,65 @@ and eval_person_field_var conf base env ((p, p_auth) as ep) loc = function
       match get_env "cnt" env with
       | Vint cnt -> VVstring (string_of_int cnt)
       | _ -> VVstring "")
-  | [ "cous_paths_min_date"; l1; l2 ] ->
+  | [ "cous_paths_min_date"; l1; l2 ] -> (
       let _cousins_cnt, cousins_dates =
         match (!cousins_t, !cousins_dates_t) with
         | Some t, Some d_t -> (t, d_t)
         | _, _ -> init_cousins_cnt conf base env p
       in
-      let i = try int_of_string l1 with Failure _ -> raise Not_found in
-      let j = try int_of_string l2 with Failure _ -> raise Not_found in
-      let list = cousins_dates.(i).(j) in
-      if List.length list > 0 then str_val (string_of_int (List.hd list))
-      else null_val
-  | [ "cous_paths_max_date"; l1; l2 ] ->
+      match int_of_string_opt l1, int_of_string_opt l2 with
+      | Some i, Some j -> (
+          let l = cousins_dates.(i).(j) in
+          match List.length l with
+          | 0 -> null_val (* this case is legit *)
+          | _ -> str_val (string_of_int (List.nth l 0)))
```

Comment:
you can match on `l` directly and not bother with the length:

```ocaml
match l with
| [] -> null_val (* this case is legit *)
| x::_l -> str_val x
```

Reply:
done